### ISSUE TITLE
blog theme

### ISSUE THREAD

**Issue by liangpeili** [channikhabra/channikhabra.github.com#1]
Hi, I really like the theme of your blog. Can I ask which theme are you using?

**Comment by bitspook** [comment]
Hey,

It makes me very happy to hear that you liked the theme. It is a result of me fumbling with web design for the first time.  

https://www.figma.com/file/GabblehCHEF1tkyKudDnhUJ4/BrandI?node-id=0%3A1

I created it long back when I first found Figma and wanted to try my hands on designing something myself. Coincidentally my whim to design something coincided with an [initiative](https://bitspook.in/talks/personal-brands-for-software-developers-talk/index.html) I started with my work team to help them build a stronger online presence. Result was me starting [brandi](github.com/bitspook/brandi/), for which I designed this theme.

Unfortunately I got too busy with work and then distracted with other things to make much progress on it afterwards. Feel free to checkout [source branch](https://github.com/bitspook/bitspook.github.io/tree/source) of this repo and copy it over to your favorite blogging platform. It isn't complete though, so maybe complement it with something like [sakura](https://github.com/oxalorg/sakura/).

I have lots of ideas for brandi, which I hope I will get to explore now that I am on a break from work. It pains me to see my projects I once held so dear in such abandoned state.
